package.json

```diff
@@ -34,9 +34,9 @@
     "layer-check": "flub check layers --info layerInfo.json",
     "lint": "pnpm run --no-sort --stream -r lint",
     "lint:fix": "pnpm run -r --no-sort --stream lint:fix",
-    "policy-check": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json -D assert-short-codes -D npm-package-json-prettier",
-    "policy-check:asserts": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json -d assert-short-codes --fix",
-    "policy-check:fix": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json -D assert-short-codes -D npm-package-json-prettier --fix",
+    "policy-check": "flub check policy",
+		"policy-check:asserts": "flub generate assertTags --all && npm run format",
```

Comment:
minor nit - LTS is using spaces in package.json instead of tab (feel free to resolve as wont fix if an automated tool should correct this for us automatically or if we don't have any policy reqs right now on tabs v spaces)